Latex/usfmanus.cls

```diff
@@ -872,7 +872,9 @@ University of South Florida Manuscript Class]
   \newpage
 }
 
-\AtBeginDocument{
+% \AtBeginDocument{
+\RequirePackage{etoolbox}
```

Comment:
There is already a `\RequirePackage{}` [here](https://github.com/jailby/dissertation_template/blob/master/Latex/usfmanus.cls#L8)

Does it still compile for you if you do `\RequirePackage{setspace,etoolbox}`

Reply:
Yes can be moved there.

Reply:
Do you mind making the modification and committing it? To keep the class simpler

Reply:
Sure.